SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialOutputDevice.cpp

```diff
@@ -7,7 +7,7 @@
 FSpatialOutputDevice::FSpatialOutputDevice(USpatialWorkerConnection* InConnection, FString LoggerName, int32 InPIEIndex)
```

Comment:
Why are we passing it an `FString` to begin with? Let's change this to `FName` as well.